library/common-library/templates/_custom-attributes.tpl

```diff
@@ -0,0 +1,29 @@
+{{/*
+This function allows easily to overwrite custom attributes to the function "common.customAttributes"
+*/}}
+{{- define "common.customAttributes.overrideAttributes" -}}
+{{- end }}
```

Comment:
I am guessing that things like the `clusterName` would be put here. Is this correct?

On the other hand, I wonder if we need this or would just be better to trust users to ` | fromYaml | mergeOverwrite` on their side.

Reply:
Exactly that use case. This is for `nri-kubernetes` with the `clusterName` custom attribute.

Reply:
Then I'd ask the question again: Why not expect chart writers to merge this on their side?

Reply:
If there is in the `common-library` then it makes sense to have it in a separate helpers file like `nri-kubernetes` (example [here](https://github.com/newrelic/nri-kubernetes/pull/383/files#diff-3dc1d0b38b4813f0520e3397a987711789b05548c1fc55680d8bee2f3bc18699)) and makes it easier to follow up.

Also, we should/will test that override helper in a near future with some kind of chart for unit testing.

Reply:
I'm not a huge fan of the approach being different from the `agentConfig` helpers, but as long as it's documented, I'm OK with it.